src/place-screen/components/EstimatedCallItem.tsx

```diff
@@ -108,16 +111,20 @@ export const EstimatedCallItem = memo(
                 {lineName}
               </ThemeText>
             </View>
-            {mode !== 'Favourite' && <DepartureTime departure={departure} />}
-            {mode !== 'Map' && (
-              <FavouriteDepartureToggle
-                existingFavorite={existingFavorite}
-                onMarkFavourite={() =>
-                  onPressFavorite(departure, existingFavorite)
-                }
-              />
+            {mode !== 'Favourite' && (
+              <DepartureTime accessible={false} departure={departure} />
             )}
           </View>
+
+          {mode !== 'Map' && (
```

Comment:
If not a Map mode, then is Favourite? and if not a Favourite is, then a Map? not sure if we could change the conditionals to understand which component is associated with what.

Reply:
its the fact that the component is used several places, and the star is not to be shown in all of them